composer.json

```diff
@@ -44,17 +44,18 @@
     "require-dev": {
         "enqueue/amqp-lib": "^0.8",
         "friendsofsymfony/rest-bundle": "^2.1",
-        "guzzlehttp/guzzle": "^3.8",
         "jms/serializer-bundle": "^1.0 || ^2.0",
         "liip/monitor-bundle": "^2.0",
         "nelmio/api-doc-bundle": "^2.4",
+        "php-http/httplug-bundle": "^1.10",
```

Comment:
Not sure if we still need HTTPlug for this. Symfony is planing to provide an [http-client-implementation](https://github.com/symfony/http-client/blob/master/composer.json#L18) and we could use the `psr/http-client` interface instead.

Reply:
We should use `symfony/http-client`